packages/react/components/list/context/ListContext.tsx

```
@@ -0,0 +1,9 @@
+import React from 'react'
+import { ListContextProps } from './ListContextProps'
+
+export const ListContext = React.createContext<ListContextProps>({
+  divider: false,
+  ordered: false,
+  chilIndexes: [],
+  setChildIndexes: () => {},
```

ESLint

[error] Unexpected empty method 'setChildIndexes'.

(@typescript-eslint/no-empty-function)